src/odemis/driver/test/orsay_test.py

```diff
@@ -0,0 +1,652 @@
+        try:
+            cls.oserver = orsay.OrsayComponent(**CONFIG_ORSAY)
+        except Exception as e:
+            raise HwError(str(e))
+
+        cls.datamodel = cls.oserver.datamodel
+
+        for child in cls.oserver.children.value:
+            if child.name == CONFIG_PRESSURE["name"]:
+                cls.pressure = child
+
+    @classmethod
+    def tearDownClass(cls):
+        """
+        Terminate the Orsay client
+        """
+        cls.oserver.terminate()
+
+    def test_valve(self):
+        """
+        Test for controlling the gate valve of the chamber
+        """
+        self.pressure._gate.IsOpen.Target = 1
+        sleep(1)
+        self.assertTrue(self.pressure.gateOpen.value)
+
+        self.pressure._gate.IsOpen.Target = 2
+        sleep(1)
+        self.assertFalse(self.pressure.gateOpen.value)
+
+        self.pressure.gateOpen.value = True
+        self.assertEqual(self.pressure._gate.IsOpen.Target, 1)
+
+        self.pressure.gateOpen.value = False
+        self.assertEqual(self.pressure._gate.IsOpen.Target, 2)
+
+    def test_vacuum_sim(self):
+        """
+        Test for controlling the vacuum that can be run in simulation and on the real system
+        """
+        self.pressure.moveAbs({"vacuum": 1}, wait=False)
+        sleep(1)
+        self.pressure.stop()
+        self.assertEqual(int(self.pressure._chamber.VacuumStatus.Target), 1)
+
+        self.pressure.moveAbs({"vacuum": 2}, wait=False)
+        sleep(1)
+        self.pressure.stop()
+        self.assertEqual(int(self.pressure._chamber.VacuumStatus.Target), 2)
+
+        self.pressure.moveAbs({"vacuum": 0}, wait=False)
+        sleep(1)
+        self.pressure.stop()
+        self.assertEqual(int(self.pressure._chamber.VacuumStatus.Target), 0)
+
+    def test_vacuum_real(self):
+        """
+        Test for controlling the real vacuum
+        """
+        if TEST_NOHW == "sim":
+            self.skipTest("TEST_NOHW is set, cannot change vacuum pressure in simulation")
+
+        f = self.pressure.moveAbs({"vacuum": 1})
+        f.wait()
+        self.assertEqual(self.pressure.position.value["vacuum"], 1)
+        self.assertAlmostEqual(self.pressure.pressure.value, 50000, delta=5000)  # tune the goal and alowed difference
+
+        f = self.pressure.moveAbs({"vacuum": 2})
+        f.wait()
+        self.assertEqual(self.pressure.position.value["vacuum"], 2)
+        self.assertAlmostEqual(self.pressure.pressure.value, 0.1, delta=0.01)  # tune the goal and alowed difference
+
+        f = self.pressure.moveAbs({"vacuum": 0})
+        f.wait()
+        self.assertEqual(self.pressure.position.value["vacuum"], 0)
+        self.assertAlmostEqual(self.pressure.pressure.value, 100000, delta=10000)  # tune the goal and alowed difference
+
+        self.pressure.moveAbs({"vacuum": 1})
+        f = self.pressure.moveAbs({"vacuum": 0})
+        f.wait()
+        self.assertEqual(self.pressure.position.value["vacuum"], 0)
```

Comment:
All these "0 Pa" are probably not going to be acceptable on the real system (it's unreachable). We should either find typical pressure values now, or review all these numbers once we have access to the real hardware. If you want to wait, it's fine, but make it a big "TODO".